subprojects/core/src/main/java/org/gradle/process/internal/worker/request/Receiver.java

```diff
@@ -37,6 +39,7 @@
  * <p>
  * This receiver is used per worker action.
  */
+@NonNullApi
```

Comment:
Added, as the `ResponseProtocol` became also `@NonNullApi`.

Reply:
👍